app/src/main/kotlin/com/simplemobiletools/calendar/pro/dialogs/ColorPickerDialog.kt

```diff
@@ -0,0 +1,247 @@
+package com.simplemobiletools.calendar.pro.dialogs
```

Comment:
cant we reuse the dialog from Commons? Maybe with some extra checks or arguments?

Reply:
for some reason, I didn't realize it could be used by other apps too, here is the commons PR: https://github.com/SimpleMobileTools/Simple-Commons/pull/1675